CHANGELOG.md

```diff
@@ -39,6 +94,7 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 - Added accessibility complaint form.
 - Added "File an EEO Issue" form.
 - Added `/offices/office-of-civil-rights/` page, tests, and link in footer.
+- Added time macro.
```

Comment:
Changelog needs an update (two entries for this). I'll fix and push directly to spring PR before merging.